docs/pages/careers.tsx

```diff
@@ -395,58 +416,40 @@ function CareersContent() {
               theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
           }}
         />
-        <Typography
-          component="h3"
-          variant="h5"
-          color="primary"
-          fontWeight="extraBold"
-          sx={{ mb: 2 }}
-        >
-          Engineering
-        </Typography>
-        {openRolesData.engineering.map((role) => (
-          <React.Fragment>
-            <Role title={role.title} description={role.description} url={role.url} />
+        <Stack
+          spacing={2}
+          divider={
             <Divider
               sx={{
                 my: { xs: 1, sm: 2 },
                 borderColor: (theme) =>
                   theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
               }}
             />
-          </React.Fragment>
-        ))}
-        <Typography
-          component="h3"
-          variant="h5"
-          color="primary"
-          fontWeight="extraBold"
-          sx={{ mb: 2 }}
+          }
         >
-          Product
-        </Typography>
-        {openRolesData.product.map((role, idx, arr) => (
-          <React.Fragment>
-            <Role title={role.title} description={role.description} url={role.url} />
-            {idx < arr.length - 1 && (
-              <Divider
-                sx={{
-                  my: { xs: 1, sm: 2 },
-                  borderColor: (theme) =>
-                    theme.palette.mode === 'dark' ? 'primaryDark.600' : 'grey.100',
-                }}
-              />
-            )}
-          </React.Fragment>
-        ))}
+          {Object.values(openRolesData).map((category) => {
```

Comment:
Looks like you never use their keys so you might as well use an array to ensure IE11 compat. Bad look in general if people audit IE11 compat.

Reply:
Yeah ok, we don't need an object, an array is good enough, I'm updating.

I would vote for ignoring IE 11 on the website and the docs. I have just tried on BrowserStack, all the pages are broken on http://mui.com/, docs included.